tidb-cloud/secure-connections-to-serverless-tier-clusters.md

```diff
@@ -1,50 +1,28 @@
 ---
-title: Secure Connections to Serverless Tier Clusters
-summary: Introduce TLS connection in TiDB Serverless Tier clusters.
+title: TLS connects to Serverless Tier
```

Comment:
How about `TLS connection to Serverless Tier`?

Reply:
Need a simple theme name, maybe this is a good one

Reply:
"TLS connects to Serverless Tier" does not look accurate from the writing perspective,  "TLS connection to Serverless Tier" looks fine.